config/environments/test.rb

```diff
@@ -41,9 +41,4 @@
 
   # Print deprecation notices to the stderr
   config.active_support.deprecation = :stderr
-
-  if ENV["RAILS_STDOUT_LOGGING"].present?
-    # Disable logging to file. It might have performance impact while using Docker for Mac (slow filesystem sync).
-    config.logger = Logger.new(STDOUT)
-  end
```

Comment:
I suppose this means when you are running the tests locally you might end with large log files that are being synced and also saved in the docker container. But I guess if that isn't causing you problems yet, then we can deal with it later.

To deal with the syncing part I'd modify this line: https://github.com/concord-consortium/docker-volume-sync/blob/master/entrypoint.sh#L46 so we can pass additional `-ignore` options via a environment variable to unison.  So then we can configure the sync container to ignore the log folder.

Reply:
> But I guess if that isn't causing you problems yet, then we can deal with it later.

Right, I think the problem starts with huge log files (like my old ones that had a few GB). I didn't have better idea of how to quickly fix it. But with STDOUT logging tests were not usable, as results were totally lost in SQL statements.